app/assets/javascripts/admin/line_items/controllers/line_items_controller.js.coffee

```diff
@@ -11,54 +12,76 @@ angular.module("admin.lineItems").controller 'LineItemsCtrl', ($scope, $timeout,
   $scope.sharedResource = false
   $scope.columns = Columns.columns
   $scope.sorting = SortOptions
+  $scope.distributorFilter = ''
+  $scope.supplierFilter = ''
+  $scope.orderCycleFilter = ''
+  $scope.quickSearch = ''
+  $scope.page = 1
+  $scope.per_page = 50
 
   $scope.confirmRefresh = ->
     LineItems.allSaved() || confirm(t("unsaved_changes_warning"))
 
   $scope.resetSelectFilters = ->
-    $scope.distributorFilter = 0
-    $scope.supplierFilter = 0
-    $scope.orderCycleFilter = 0
-    $scope.quickSearch = ""
+    $scope.distributorFilter = ''
+    $scope.supplierFilter = ''
+    $scope.orderCycleFilter = ''
+    $scope.quickSearch = ''
+    $scope.page = 1
+    $scope.refreshData()
+
+  $scope.changePage = (newPage) ->
+    $scope.page = newPage
+    $scope.refreshData()
 
   $scope.refreshData = ->
-    unless !$scope.orderCycleFilter? || $scope.orderCycleFilter == 0
-      $scope.startDate = moment(OrderCycles.byID[$scope.orderCycleFilter].orders_open_at).format('YYYY-MM-DD')
-      $scope.endDate = moment(OrderCycles.byID[$scope.orderCycleFilter].orders_close_at).startOf('day').format('YYYY-MM-DD')
+    unless !$scope.orderCycleFilter? || $scope.orderCycleFilter == ''
+      start_date = OrderCycles.byID[$scope.orderCycleFilter].orders_open_at
+      end_date = OrderCycles.byID[$scope.orderCycleFilter].orders_close_at
+      format = "YYYY-MM-DD HH:mm:ss Z"
+      $scope.startDate = moment(start_date, format).format('YYYY-MM-DD')
+      $scope.endDate = moment(end_date, format).startOf('day').format('YYYY-MM-DD')
 
     formatted_start_date = moment($scope.startDate).format()
     formatted_end_date = moment($scope.endDate).add(1,'day').format()
 
+    return unless moment(formatted_start_date).isValid() and moment(formatted_start_date).isValid()
```

Comment:
this looks like a copy paste error, the second part would test formatted_end_date, right?